network_node.go

```diff
@@ -47,6 +52,8 @@ type BasicNetworkNode struct {
 	Network        *BasicVideoNetwork
 	outStreams     map[peer.ID]*BasicOutStream
 	outStreamsLock *sync.Mutex
+	sign           func(data []byte) ([]byte, error)
+	verify         func(data []byte, sig []byte, strmID string) bool
```

Comment:
Can we set a default "verify" here that just returns true?  This way the default behavior is `pass`.

Reply:
Generally I don't like enabling insecure defaults, but in this case we don't have a choice until the Livepeer client can actually do verification. If we can remove the default behavior after client support is in, that'd be OK.

Reply:
Fixed in https://github.com/livepeer/go-livepeer-basicnet/pull/32/commits/bdb52f96dab751294da9bce3fd2a5ba0751dcbfc